devtools/datasette/publish.py

```diff
@@ -85,12 +90,16 @@ def inspect_data(datasets: list[str], pudl_output: Path) -> str:
 
 @click.command(context_settings={"help_option_names": ["-h", "--help"]})
 @click.option(
-    "--fly",
-    "-f",
+    "--production",
     "deploy",
-    flag_value="fly",
-    help="Deploy Datasette to fly.io.",
-    default=True,
```

Comment:
Got rid of the default here to make it harder to accidentally deploy to production.